fastlane/swift/Fastlane.swift

```diff
@@ -1362,6 +1362,7 @@ func frameScreenshots(white: String? = nil,
                       forceDeviceType: String? = nil,
                       useLegacyIphone5s: Bool = false,
                       useLegacyIphone6s: Bool = false,
+                      forceOrientationBlock: String = "#<Proc:0x007fabae01e508@/Users/felixkrause/Developer/fastlane/frameit/lib/frameit/options.rb:47>",
```

Comment:
Seems like something is wrong with the Swift generation here

Reply:
Gotta set that parameter type as a block in the options.rb, like the “on_error_callback”. I’m not totally confident the api will be generated properly even with that though, since the generation assumes the block must accept a string and return nothing.